docs/contact.html

```diff
@@ -28,7 +28,7 @@
          <div class="col-md-3"></div>
          <div class="col-md-6 small-dayof-title">
             <div>
-              <span>June 23rd, 2023 - De Doelen, Rotterdam</span>
+              <span>Coming back in 2024 - De Doelen, Rotterdam</span>
```

Comment:
The date is already known, right? Why not already include it!

Reply:
Eens.

Reply:
I don't think we have publicly communicated that date, right?

I figured we would update the site at the same time that we send the "save the date" newsletter, and at that moment put the date on the site.